src/feature/hs/hs_client.c

```diff
@@ -1778,6 +1778,16 @@ hs_client_circuit_cleanup_on_free(const circuit_t *circ)
                                      &orig_circ->hs_ident->intro_auth_pk,
                                      failure);
     break;
+  case CIRCUIT_PURPOSE_C_REND_READY_INTRO_ACKED:
+    log_info(LD_REND, "Failed v3 intro circ for service %s to intro point %s "
+                      "(rendevous point failed).",
+        safe_str_client(ed25519_fmt(&orig_circ->hs_ident->identity_pk)),
+        safe_str_client(build_state_get_exit_nickname(
+                                                     orig_circ->build_state)));
```

Comment:
Hmmm, this circuit is a rendezvous circuit, not the introduction circuit so you are logging and flagging the wrong circuit here.